docs/configuration/remote-admin.mdx

```diff
@@ -6,6 +6,10 @@ sidebar_position: 3
 description: An advanced feature which allows remote administration of a device through a secure channel on the Mesh instead of via Bluetooth, Serial, or IPv4.
 ---
 
+:::warning Out of Date
+This page is out of date for firmware 2.5+ and will not work.
+:::
+
```

Comment:
A warning here seems fair as a short term fix, but perhaps we could word it a little differently?  How about something like:

:::info
Since the introduction of improved PKI-based administration in firmware 2.5, using the legacy admin channel described on this page is no longer supported.
:::

Reply:
I'm in the middle of rewriting this page. I'll try and include a suitable signpost when I get to that bit.